student.rb

```diff
@@ -1,9 +1,11 @@
 require_relative 'person'
 
 class Student < Person
-  def initialize(age, classroom, name: 'Unknown', parent_permission: true)
+  attr_accessor :classroom
+
+  def initialize(age, name: 'Unknown', parent_permission: true)
     super(age, name: name, parent_permission: parent_permission)
-    @classroom = classroom
+    @classroom = nil
   end
```

Comment:
- [ ] Please add a method that serves as the `belongs-to` side (a student belongs to a classroom). You can use something like the following:
```ruby
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
  ```